src/common/styles/cd-filler-loading.less

```diff
@@ -1,3 +1,18 @@
+.cd-filler {
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+  position: relative;
+  &::before {
+    content: " ";
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    animation: fillerLoading .8s linear infinite;
+    background: linear-gradient(to right, transparent 42%, fade(@cd-white, 50%) 50%, transparent 58%) 50% 50%;
+  }
+}
+
 .cd-filler-loading() {
```

Comment:
This doesn't need to be a function anymore, and you definitely don't need to call it in cd-event-sessions.vue anymore. Just import the less file and you should be good!

Reply:
:+1: